analyses/telomerase-activity-prediction/06-TERTp-EXTENDScores.R

```diff
@@ -59,6 +77,39 @@ metadata_df <- read_tsv(metadata_file, guess_max = 10000)
 maf_df <- read_tsv(maf_file)
 hotspot_df <- read_tsv(hotspot_file)
 
+# Read in and export TERT SV data in BED format for knotAnnotSV to assess pathogenicity of the SV call
+# This BED file was put into https://lbgi.fr/AnnotSV/ and the TSV downloaded for ACMG assessment below
+sv_df <- read_tsv(sv_file) %>%
+  # only keep PASS variants
+  filter(FILTER == "PASS",
+         # let's check all 
+         grepl("TERT", Gene.name)) %>%
+  select(chrom = SV.chrom, chromStart = SV.start, chromEnd = SV.end, name = AnnotSV.ID, 
+                  SVtype = SV.type, Sample = Kids.First.Biospecimen.ID.Tumor) %>%
+  write_tsv(file.path(results_dir, "mantaSV_tert.bed"))
```

Comment:
Can we move this and subsequent file definitions to top of this R script so they aren't hardcoded within `write_tsv` lines?

Reply:
I am removing this now